lerobot/common/policies/act/modeling_act.py

```diff
@@ -273,14 +287,6 @@ def _reset_parameters(self):
     def forward(self, batch: dict[str, Tensor]) -> tuple[Tensor, tuple[Tensor, Tensor] | tuple[None, None]]:
         """A forward pass through the Action Chunking Transformer (with optional VAE encoder).
 
-        `batch` should have the following structure:
-
-        {
-            "observation.state": (B, state_dim) batch of robot states.
-            "observation.images": (B, n_cameras, C, H, W) batch of images.
-            "action" (optional, only if training with VAE): (B, chunk_size, action dim) batch of actions.
-        }
-
```

Comment:
FYI I removed this is it already had one inaccuracy in it, I would have needed to update it for this PR, and I determined that it is no longer worth the maintenance overhead. I think one can easily gather these things from higher level / upstream code. Lmk if you disagree.

Reply:
Having the information there was pretty useful I think, could we keep it? and add something like
```"observation.environment_state"` : (B, nb_keypoints) ```

Reply:
I restored it and updated it.